### DISCUSSION TITLE
How to use tailwindcss with nicegui ?

### DISCUSSION THREAD

**Discussion by muratbedir** [zauberzeug/nicegui#1881] [Q&A]
### Question

When look tailwindcss documentation  found below class values how I can use these values with nicegui ? I try to .classes it not work, then I try to .style but can not work. Is there any suggestions. 

<button class="bg-sky-500 hover:bg-sky-700 ...">
  Save changes
</button> 

![image](https://github.com/zauberzeug/nicegui/assets/48950062/d2214931-3882-44d5-8b3e-ab4358995339)

**Comment by falkoschindler** [comment]
You can use `.classes` or the [`.tailwind`](https://nicegui.io/documentation#tailwind_css) API.
Please be more specific and share a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example) of what doesn't work.
